docs/pages/admin-guides/management/guides/aws-iam-identity-center.mdx

```diff
@@ -182,11 +173,60 @@ Clicking the "Log In" button for this app takes you to your Identity Center
 SSO start page which you can use to pick a role and connect to your AWS account
 as usual.
 
+## Background: Account Assignments
+
+The main resource that the Teleport Identity Center integration manages is the
+*Account Assignment*.
+
+An Account Assignment is the combination of a specific Permission Set on a specific
+AWS account - for example "*AdminAccess on Production*" (where *Production* is 
+an AWS account managed by Identity Center). 
+
+When a user has access to an Account Assignment in Teleport, that access is
+mirrored in AWS Identity Center. When a teleport user loses access to an Account
+Assignment in Teleport, that access is similarly deleted in AWS. 
+
+Access to Account Assignments can be granted via Teleport Roles, either directly
+to Users or through Access Lists, or by Account Assignment resources included in
+an approved Access Request.
+
 ## Usage scenarios
 
 Let's take a look at some common usage scenarios enabled by the Identity Center
 integration.
 
+### Just-In-Time Access with resource Access Requests
```

Comment:
Right now the section ordering looks like below: 
- Just-In-Time Access with resource Access Requests
- Managing access with Access Lists
- Just-in-time access with role Access Requests
- Long-term access with Access Requests

Can we group and sort based on the  usage scenarios: 

- Just-In-Time Access with resource Access Requests
- Access Request
  - Just-in-time access with role Access Requests
   - Long-term access with Access Requests
   - Long-term access with Access Requests